packages/api/src/hooks/useCreateNewRealAccount.ts

```diff
@@ -0,0 +1,28 @@
+import { useMemo } from 'react';
+import useMutation from '../useMutation';
+import useInvalidateQuery from '../useInvalidateQuery';
+
+/** A custom hook that creates a new real trading account. */
+const useCreateNewRealAccount = () => {
+    const invalidate = useInvalidateQuery();
+    const { data, ...rest } = useMutation('new_account_real', {
+        onSuccess: () => {
+            invalidate('authorize');
+        },
+    });
```

Comment:
Just a question, what does this `useMutation` hook do here?

Reply:
In basic terms sends request to the API.

Reply:
https://tanstack.com/query/v4/docs/react/guides/mutations

Reply:
Alright thanks!